### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [Haynesf1/TeaCozy#3]
https://github.com/Haynesf1/TeaCozy/blob/a6c7d00fd0d73fb5762cdfe30cd8f598bdf091c3/index.html#L34

Should be using alt text for each image.  Without it, your HTML doesn't pass validation.  More importantly, alt text helps the visually impaired through their use of screen readers.
